README.md

```diff
@@ -112,49 +113,112 @@ ebcli rest-server --trust-node
 An api collection for Postman (https://www.getpostman.com/) is provided [here](./docs/peggy.postman_collection.json) which documents some API endpoints and can be used to interact with it.
 Note: For checking account details/balance, you will need to change the cosmos addresses in the URLs, params and body to match the addresses you generated that you want to check.
 
-## Running the relayer service
+## Running the bridge locally
+
+With the application set up, you can now use Peggy by sending a lock transaction to the smart contract.
+
+### Set-up
+
+Create a .env file with environment variable LOCAL_PROVIDER - an example configuration can be found in .env.example. For running the bridge locally, you'll only need the LOCAL_PROVIDER. Environment variables MNEMONIC and INFURA_PROJECT_ID are only required for using the Ropsten testnet (see below).
+
+### Terminal 1: Start local blockchain
+
+```
+$ cd testnet-contracts/
```

Comment:
remove $ signs from commands please, they prevent the user from being able to copy paste

Reply:
https://github.com/swishlabsco/peggy/pull/2/commits/acd43659366cc251c0bc42c77141a5e2c05e78f8